src/mempool.rs

```diff
@@ -215,11 +215,14 @@ impl Tracker {
             })
             .collect();
         if !entries.is_empty() {
-            let txids: Vec<&Txid> = entries.iter().map(|(txid, _)| *txid).collect();
-            let txs = match daemon.gettransactions(&txids) {
+            let txs = match entries
+                .iter()
+                .map(|(txid, _)| daemon.gettransaction(txid, None))
+                .collect::<Result<Vec<_>>>()
+            {
                 Ok(txs) => txs,
                 Err(err) => {
-                    debug!("failed to get transactions {:?}: {}", txids, err); // e.g. new block or RBF
+                    debug!("failed to get {} transactions: {}", entries.len(), err); // e.g. new block or RBF
```

Comment:
Note that I see this error occasionally - fetching the full mempool takes time and RBF isn't uncommon anymore. It may be worth handling it better in a future PR.